src/dlgTriggerEditor.cpp

```diff
@@ -7214,6 +7214,7 @@ void dlgTriggerEditor::slot_color_trigger_fg()
     if( ! pT ) return;
 
     QPushButton * pB = (QPushButton *) sender();
```

Comment:
:thought_balloon: Future work - "nuke C style casts" - this should be a `qobject_cast<T>` or failing that a `dynamic_cast<T>`...!